engine-protocol/src/main/proto/engine_control.proto

```diff
@@ -0,0 +1,64 @@
+syntax = 'proto3';
+package engine_protocol;
+
+option java_multiple_files = false;
+option java_package = "io.camunda.zeebe.process.test.engine.protocol";
+
+message StartEngineRequest {}
+
+message StartEngineResponse {}
+
+message StopEngineRequest {}
+
+message StopEngineResponse {}
+
+message ResetEngineRequest {}
+
+message ResetEngineResponse {}
+
+message IncreaseTimeRequest {
+  // the amount of milliseconds the engine should increase in time
+  int32 milliseconds = 1;
+}
+
+message IncreaseTimeResponse {}
+
+message WaitForIdleStateRequest {
+  // timeout (in ms). The request will be closed if an idle state has not been
+  // achieved withing the timeout.
+  int64 timeout = 1;
```

Comment:
❓ Do we only want to have a timeout on this specific message?

Reply:
Yes. This is the only request where we will be waiting for something on the server-side. We don't want to get stuck so a timeout here seems like a good addition. This timeout is not the request timeout, it's the waiting timeout. gRPC can handle request timeouts itself.